Ginger/Ginger/Agents/AgentDriverConfigPage.xaml.cs

```diff
@@ -45,6 +47,23 @@ public AgentDriverConfigPage(Agent agent, General.eRIPageViewMode viewMode = Gen
             InitializeComponent();
 
             mAgent = agent;
+
+            if (Agent.GetDriverPlatformType(mAgent.DriverType).Equals(ePlatformType.Web) && !mAgent.DriverConfiguration.Any((driverConfig)=>string.Equals(nameof(SeleniumDriver.ByPassProxyAddresses), driverConfig.Parameter)))
```

Comment:
instead of manually changing the position, check if there is another way where it will be on second position while it was getting serialized to xml, if not, lets skip it for now and remove this code

Reply:
This code is for backward compatibility, when the existing web agent is loaded in the latest ginger version